source/common/network/address_impl.cc

```diff
@@ -47,14 +48,14 @@ Address::InstanceConstSharedPtr addressFromSockAddr(const sockaddr_storage& ss,
   NOT_REACHED;
 }
 
-InstanceConstSharedPtr addressFromFd(int fd) {
+InstanceConstSharedPtr addressFromFd(int fd, bool v6only) {
```

Comment:
In this case, shouldn't we getsockopt to determine if it is set, instead of having this as a parameter?

Reply:
Yep, this makes sense.